zuul.d/project-templates.yaml

```diff
@@ -457,6 +457,7 @@
         - ansible-test-sanity-docker-stable-2.10
         - ansible-test-sanity-docker-stable-2.11
         - ansible-test-units-cloud-common-python38
+        - build-ansible-collection
```

Comment:
this won't actually build artifacts, as there are no projects to build.

Reply:
I just need the cloud.common artifact. I thought this was enough for this case.

Reply:
This is what is actually done in the `publish-to-galaxy-master-only` template.

Reply:
yes, that is correct. However that would use the released artifacts on galaxy.a.c. however, if you are doing periodic testing, you'd want to ensure the latest code from git is used likely, not released?

Reply:
I would like to let build-ansible-collection prepare the tarball of the collection. This way the jobs can run properly.